interface/src/app/analyze/experiment/experiment.js

```diff
@@ -9,11 +9,7 @@ angular.module('adage.analyze.experiment', [
     '/api/v0/experiment/:accession/',
     // TODO need to add logic for handling pagination of results.
     // then, can change "limit" below to something sensible
-    {accession: '@accession', limit: 0},
-    // Angular expects a query service to give only a list but our Tastypie
-    // interface wraps the response list with pagination so we tell Angular to
-    // expect an object instead via isArray: false
-    {'get': {method: 'GET', isArray: false}}
+    {limit: 0}
```

Comment:
Right. I learned from @dongbohu here that the `@var` syntax does not apply for `GET` requests so it was doing nothing.

In addition, the `isArray: false` was something I needed for `query` calls (see the comment above) but does not apply to `get`, as we have here.